src/utiltime.cpp

```diff
@@ -25,6 +25,12 @@ int64_t GetTime()
     return now;
 }
 
+int64_t GetMockableTimeMicros()
+{
+    if (nMockTime) return nMockTime * 1000000;
```

Comment:
Clever; I like it. At some point I can imagine wanting finer-grained precision, but that's not now, so I'm leaving this comment here as a breadcrumb that we deferred that decision.